content/docs-beta/SUSHI/project/_index.md

```diff
@@ -59,7 +59,7 @@ You can populate your project as follows:
 
 * **sushi-config.yaml**: This file provides project configuration data to SUSHI. It is described further in the [Configuration](/docs-beta/sushi/configuration/) documentation.
 * **input/fsh/\*.fsh**: FSH files contain the FHIR Shorthand definitions for all the resources and examples in your IG.
-* **ig.ini**: Configuration file required for the FHIR IG Publication process.
+* **ig.ini**: Configuration file required for the FHIR IG Publication process. NOTE: This file MUST use the `#current` version of the `fhir.base.template` (or a template that extends this version). So to use the `fhir.base.template`, the `template` property in **ig.ini** will be `template = fhir.base.template#current`.
```

Comment:
I think we should mention that this is a temporary limitation and we should be more explicit about templates allowed.  How about this?
```
* **ig.ini**: Configuration file required for the FHIR IG Publication process. NOTE: As of the SUSHI 1.0 Beta release, this file MUST use a template based on `fhir.base.template#current`. Specific template versions (i.e., other than `#current`) are expected to work in the future.  For now, any of the following should work:
  * `template = fhir.base.template#current`
  * `template = hl7.base.template#current`
  * `template = hl7.fhir.template#current`
  * `template = hl7.davinci.template#current`
  * `template = hl7.cda.template#current`
```

Reply:
I also think we should have a note about this at the top of the page.  I just want to make sure people are really aware of it, as I expect it to trip folks up.